src/web/.env.example

```diff
@@ -1,7 +1,7 @@
 # Environment variable overrides for local development
 FLASK_APP=autoapp.py
 FLASK_ENV=development
-DATABASE_URL="sqlite:////tmp/dev.db"
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432"
```

Comment:
Is there some hard reason why to change the backend away from sqlite?

Reply:
It is possible to treat it as a plain string, but I would have to dump/load it each time manually, and no search possibility. And we would still have to switch to some DBMS, because sqlite is not the best thing for production, I believe.

Reply:
I would like the DB platform to be `sqlalchemy`, i.e. DB backend agnostic in the long term. However, I am OK with the current state until we figure out how to structure the data.

Reply:
hm, my comment wasn't posted or was deleted somehow, will repeat it. 
We're still using sqlalchemy with postgresql. Postgresql brings only one dependency - json field, but it also can be replaced with text field if needed.

Reply:
Yes, I agree, but it is not possible any longer to choose a different backend. Although it is common for most projects, TEAP is so lightweight that it is IMO possible to aim for it to be DB-agnostic without sacrificing anything.

Reply:
Also please have `DB_HOST`, `DB_USER` and `DB_PASSWORD` as separate config items. That way, the `.env` file will play nicely with docker-compose.